### PR TITLE
Add code sample and tests for rankingScoreThreshold search parameter

### LINKED ISSUE #544: [v1.9] Support `rankingScoreThreshold` search parameter
*Following this [central issue](https://github.com/meilisearch/integration-guides/issues/301)*

This is related to a newly introduced feature in [Meilisearch v1.9.0](https://github.com/meilisearch/meilisearch/releases/tag/v1.9.0): the `rankingScoreThreshold` search parameter.

Refer to [docs for more information](https://www.meilisearch.com/docs/reference/api/search#ranking-score-threshold).

- [x] Ensure the library can accept the new `rankingScoreThreshold` search parameter.
- [ ] ⚠️ Add integration tests ⚠️
- [ ] Update the `.code-samples.meilisearch.yaml`
    - [ ]  Add `search_parameter_reference_ranking_score_threshold_1` key and “translate” the following curl example by using the newly added methods: https://github.com/meilisearch/documentation/blob/04f26a35771305378ed0a506f4d60ef06d07bd7e/.code-samples.meilisearch.yaml#L1344

### PR DESCRIPTION
**Pull request by andre-m-dev**
# Pull Request

## Related issue
Fixes #544

## What does this PR do?
rankingScoreThreshold implemented

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

### CODE REVIEW

**Comment by andre-m-dev** [comment]
I don't know why the tests for ruby 3.1 fail every time

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -653,3 +653,7 @@ update_search_cutoff_1: |-
   client.index('movies').update_search_cutoff_ms(150)
 reset_search_cutoff_1: |-
   client.index('movies').reset_search_cutoff_ms
+search_parameter_reference_ranking_score_threshold_1: |-
+  client.index('movies').search('badman', {
```
Comment (curquiza) [inline .code-samples.meilisearch.yaml:657]:
```suggestion
  client.index('INDEX_NAME').search('badman', {
```

**Review by curquiza — APPROVED** [review]
Niiice

bors merge

**Review by curquiza — APPROVED** [review]
bors merge

**Comment by curquiza** [comment]
@andre-m-dev sorry, looks like the tests does not pass since I merged your PR for distinct first 😢 

Can you fix the tests?

**Comment by andre-m-dev** [comment]
@curquiza should be fine now

**Review by curquiza — APPROVED** [review]
bors merge

**Pull request merged**